Formula/coredns.rb

```diff
@@ -0,0 +1,23 @@
+class Coredns < Formula
+  desc "Plugin-driven DNS and service discovery"
+  homepage "https://coredns.io"
+  url "https://github.com/coredns/coredns/archive/v1.3.1.tar.gz"
+  sha256 "b471e26c40140fdb05039b36cb8fb804b03ceb329d48a592cbca9ad6550c8143"
+
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/coredns/coredns").install buildpath.children
+
+    cd "src/github.com/coredns/coredns" do
+      system "make", "godeps", "all"
+      bin.install "coredns"
```

Comment:
No man page, no documentation?

Reply:
This will be documented in the official CoreDNS docs but I suppose I can include the metafiles